modules/web/js/ballerina/ast/statements/if-else-statement.js

```diff
@@ -36,6 +36,8 @@ class IfElseStatement extends Statement {
         this._ifStatement = ifStatement;
 
         this._elseStatement = undefined;
```

Comment:
we don't need this undefined assignment. No need to specially assign this to a variable.